runtime/src/native_loader.rs

```diff
@@ -46,7 +46,7 @@ fn create_path(name: &str) -> PathBuf {
     }
 }
 
-pub fn entrypoint(
+pub fn load_program_and_process_instruction(
```

Comment:
Why name the native loader's entrypoint different than other programs?

Reply:
This was a shot in the dark. I can revert.

Reply:
done